piker/ui/_interaction.py

```diff
@@ -340,6 +360,49 @@ async def handle_viewmode_mouse(
             view.order_mode.submit_order()
 
 
+class OverlayT(Struct):
```

Comment:
Note this code isn't used at all yet, it's just a draft for a more encapsulated API for the main logic inside `.interact_graphics_cycle()` below.